packages/components/psammead-social-embed/src/Canonical/index.jsx

```diff
@@ -78,8 +77,8 @@ const CanonicalEmbed = ({ provider, oEmbed, onRender }) => {
   useEffect(providers[provider].enrich);
 
   useEffect(() => {
-    if (provider === fixtures.twitter.source && isSdkLoaded && onRender) {
-      providers[fixtures.twitter.source].onSdkLoad(onRender);
+    if (provider === 'twitter' && isSdkLoaded && onRender) {
```

Comment:
Not one for this PR but looking back at the previous PR, I think it might have been clearer if `onRender` was `onTwitterEmbedRender` or something. Better to genericise it later if the functionality was added to other Embed types.

Reply:
being nitpicky but `isSdkLoaded` is like a question rather than a fact. What about renaming to `hasLoadedLibrary` or something instead?

Reply:
what about making this like:

```jsx
  useEffect(() => {
    const { onLibraryLoad } = providers[provider]

    if (hasLoadedLibrary && onLibraryLoad) {
      onLibraryLoad(onRender)
    }
  }, [hasLoadedLibrary]);
```

and making `onRender` a no-op function in default props?

Reply:
Thanks @jroebu14, this is a great idea, will make the change now 👍

Reply:
For the YouTube and Instagram `onLibraryLoad` methods you could console log an error `onRender callback function not implemented for YouTube` or something

Reply:
Yeah good idea 👍